catalog-info.yaml

```diff
@@ -1,3 +1,18 @@
+# Declare a Backstage Component that represents your application.
+---
+# yaml-language-server: $schema=https://json.schemastore.org/catalog-info.json
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: fleet-server
+  description: fleet-server - The control server for managing a fleet of elastic-agents
+
+spec:
+  type: tool
+  owner: group:ingest-fp # Find your group in https://backstage.elastic.dev/catalog?filters%5Bkind%5D=group

```

Comment:
Can you remove the comment in these lines

Reply:
done